pkg/core/blockchain.go

```diff
@@ -1117,22 +1117,21 @@ func (bc *Blockchain) getNEP17Balance(h util.Uint160, acc util.Uint160, bw *io.B
 
 // GetUtilityTokenBalance returns utility token (GAS) balance for the acc.
 func (bc *Blockchain) GetUtilityTokenBalance(acc util.Uint160) *big.Int {
-	bs, _ := bc.getNEP17Balance(bc.contracts.GAS.Hash, acc, nil) // no error while getting GAS balance
-	if bs == nil {
-		return big.NewInt(0)
-	}
-	return bs
+	return bc.getNativeNEP17Balance(bc.contracts.GAS.Hash, acc)
 }
 
 // GetGoverningTokenBalance returns governing token (NEO) balance and the height
 // of the last balance change for the account.
-func (bc *Blockchain) GetGoverningTokenBalance(acc util.Uint160) (*big.Int, uint32) {
-	bs := bc.GetNEP17Balances(acc)
+func (bc *Blockchain) GetGoverningTokenBalance(acc util.Uint160) *big.Int {
```

Comment:
Last update block index has a lot of meaning for NEO (unlike other NEP-17).

Reply:
Fixed, although we don't use it anywhere except a single place in the RPC server.